notebooks/kaggle/store-sales-time-series-forecasting/experiments/prophet.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "acb441b0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(os.environ['PROJECT_ROOT'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8fd7c34f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from prophet import Prophet\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45c31fbb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_ROOT = Path('data', 'kaggle', 'store-sales-time-series-forecasting')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "500c3ef8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train = pd.read_csv(DATA_ROOT / 'prepared_train.csv', low_memory=False)\n",
+    "train.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8df45cc3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_prepared_data = train[['date', 'sales']].rename(columns={'date': 'ds', 'sales': 'y'})\n",
+    "train_prepared_data = train_prepared_data.groupby('ds').median().reset_index()\n",
```

Comment:
Instead of a single prophet model trained on the aggregated data it will be much better to build `54*33` separate prophet models for each time-series.